src/ansiblelint/config.py

```diff
@@ -204,7 +204,18 @@ def in_venv() -> bool:
 
 def guess_install_method() -> str:
     """Guess if pip upgrade command should be used."""
+    # pylint: disable=import-outside-toplevel
+    from importlib.metadata import distribution
```

Comment:
Put it outside, no need to use this approach in this case. Only with ansible core we need these.

Reply:
ack